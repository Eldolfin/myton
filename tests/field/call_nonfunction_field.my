class Foo:

foo = Foo()
foo.bar = "not a function"

foo.bar()
