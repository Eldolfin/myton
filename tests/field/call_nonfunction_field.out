[31merror[error]: 'str' object is not callable
----- Traceback -----
  â”Œâ”€ <unknown>:6:9
  | 
6 | foo.bar()
  |          ^
[0m